js/src/attribute-mapping/attribute-mapping-table-categories.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { _n, sprintf, _x, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppTooltip from '.~/components/app-tooltip';
+
+const DUMMY_CATEGORIES = [
+	{ id: 1, name: 'Category 1' },
+	{ id: 2, name: 'Category 2' },
+	{ id: 3, name: 'Category 3' },
+];
+
+const SEPARATOR = _x(
+	', ',
+	'the separator for concatenating the categories where the Attribute mapping rule is applied.',
+	'google-listings-and-ads'
+);
+
+const CATEGORIES_TO_SHOW = 5;
+
+/**
+ * Renders a text and maybe a tooltip showing the categories for an Attribute Mapping rule
+ *
+ * @param {Object} props Component props
+ * @param {string}  props.categories The categories to render
+ * @param {string} props.condition The condition to which this categories apply
+ *
+ * @return {JSX.Element|string} The component
+ */
+const AttributeMappingTableCategories = ( { categories, condition } ) => {
+	if ( condition === 'ALL' ) {
+		return __( 'All', 'google-listings-and-ads' );
+	}
+
+	const categoryArray = categories.split( ',' );
+	const categoryNames = categoryArray
+		.slice( 0, CATEGORIES_TO_SHOW )
+		.map( ( category ) => {
+			return DUMMY_CATEGORIES.find(
+				( e ) => e.id === parseInt( category, 10 )
+			).name;
+		} )
+		.join( SEPARATOR );
+
+	const maybeDots =
```

Comment:
It would be great if the variable could be renamed as well for the respective change in https://github.com/woocommerce/google-listings-and-ads/pull/1691/commits/f3bf7a8a4cc8974d58e9de5869d6b41e76cde3d7